CODE_REVIEW.md

```diff
@@ -0,0 +1,16 @@
+# Improvements:
+
+* Website is not responsive for mobile devices, made it responsive.
+* Private member variable ‘http' is never reassigned, marked it as 'readonly’.
+* Filtering reading list API (/api/reading-list) response and passing only the list with book items, thereby avoiding any empty objects.
+* We can implement caching our HTTP requests to avoid hitting same endpoints multiple times.
+* We can add routing for search key value. Even if user reloads the page, he can land on the search item.
+* The catchError operator part should go within the switchMap pipe.
+* Added missing methods in reading-list reducer file.
+
+# Accessibility Issues:
+
+* Image element should have an "alt" attribute, it provides a textual alternative to an image.
+* Form elements like input element should have associated labels, they ensure that form elements are announced properly.
+* Use aria-label to provide an invisible label where a visible label cannot be used.
+* We can use semantic tags like section instead of just using div.
```

Comment:
There should be at least 5 items here.

Did you run Lighthouse? Can you make it clear which of these came from doing that and which you found on your own?

If you need advice on this you can contact Bobby Roe from the Accessibility Resource Center (ARC) for training resources: Bobby.Roe2@t-mobile.com

Reply:
Solved lighthouse issues but not mentioned in the list. Updated the list with lighthouse issues and their solutions.